assets/data-port/import/data/actions.js

```diff
@@ -297,21 +288,34 @@ export const errorFileUpload = ( level, error ) => ( {
 } );
 
 /**
- * @typedef  {Object} SetStepDataAction
- * @property {string} type Action type.
- * @property {string} step Step name.
- * @property {Object} data Step data.
+ * Load the import log for the given job.
+ *
+ * @param {string} jobId The job id.
  */
+export const fetchImportLog = composeFetchAction(
+	START_FETCH_IMPORT_LOG,
+	function*( jobId ) {
+		const data = yield fetchFromAPI( {
+			path: buildJobEndpointUrl( jobId, [ 'logs' ] ),
+		} );
+
+		return data;
+	},
+	SUCCESS_FETCH_IMPORT_LOG,
+	ERROR_FETCH_IMPORT_LOG
+);
```

Comment:
I added a small helper for this START, fetch, SUCCESS, ERROR pattern. It removes the individual actions creators that are mostly just noise, but still keeps the action types explicit.

Reply:
I like it.